compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolResolver.java

```
@@ -401,6 +370,88 @@ BSymbol createSymbolForStampOperator(DiagnosticPos pos, Name name, List<BLangExp
         return symTable.notFoundSymbol;
     }
 
+    BSymbol createSymbolForFromOperator(DiagnosticPos pos, Name name, BLangExpression targetTypeExpression,
+                                        List<BLangExpression> functionArgList) {
+        if (functionArgList.size() == 1) {
```

invert if condition fail fast